.github/workflows/release-automation.yml

```
@@ -0,0 +1,362 @@
+        tag_name: ${{ steps.version.outputs.version }}
+        release_name: "ETL Pipeline ${{ steps.version.outputs.version }}"
+        body: ${{ steps.changelog.outputs.changelog }}
+        draft: false
+        prerelease: ${{ github.event.inputs.prerelease == 'true' }}
+
+    - name: Upload release assets
+      uses: actions/upload-release-asset@v1
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      with:
+        upload_url: ${{ steps.create_release.outputs.upload_url }}
```

actionlint (1.7.7)

property "create_release" is not defined in object type {changelog: {conclusion: string; outcome: string; outputs: {string => string}}; version: {conclusion: string; outcome: string; outputs: {string => string}}}

(expression)